appveyor.yml

```diff
@@ -73,11 +72,12 @@
     assembly_informational_version: "2.2.0.0"
 
   build_script:
-    - msbuild "src\Vlc.DotNet.sln" /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll" /p:Configuration=Release;Platform=AnyCPU
-    - ps: nuget pack C:\projects\vlc-dotnet-master\nuget\Vlc.DotNet.Core.Interops.nuspec -version $env:APPVEYOR_BUILD_VERSION
-    - ps: nuget pack C:\projects\vlc-dotnet-master\nuget\Vlc.DotNet.Core.nuspec -version $env:APPVEYOR_BUILD_VERSION
-    - ps: nuget pack C:\projects\vlc-dotnet-master\nuget\Vlc.DotNet.Forms.nuspec -version $env:APPVEYOR_BUILD_VERSION
-    - ps: nuget pack C:\projects\vlc-dotnet-master\nuget\Vlc.DotNet.Wpf.nuspec -version $env:APPVEYOR_BUILD_VERSION
+    - msbuild /t:Restore /Verbosity:Minimal "src\Vlc.DotNet.sln"
+    - msbuild /t:Build /p:Configuration=Release "src\Vlc.DotNet.sln"
```

Comment:
Why isn't there the /logger option?

Reply:
I added it back. Not sure if it's really needed these days, though.